main.tf

```diff
@@ -1,56 +1,81 @@
 ##############################################################################
-# SCC module
+# Input variable validations
+##############################################################################
+
+locals {
+  # tflint-ignore: terraform_unused_declarations
+  validate_new_scc_instance_cos_setting = var.existing_scc_instance_crn == null && anytrue([var.cos_bucket == null, var.cos_instance_crn == null]) ? tobool("when creating a new SCC instance, both both `var.cos_instance_crn` and `var.cos_bucket` are required.") : false
```

Comment:
Pointing out a typo, of "both both" in the error message, but also could this be done using the new TF 1.9 validation? maybe on existing_scc_instance_crn variable?

Reply:
oh I didn't write that logic, I just moved it to a different line. I can fix the typo though. There is a different tracking issue to move to 1.9. I don't want to clog up this PR with other changes